test/testcafe/framework/page-objects/EnrollOktaVerifyPageObject.js

```diff
@@ -5,11 +5,19 @@ import { within } from '@testing-library/testcafe';
 const FORM_INFOBOX_ERROR_TITLE = '[data-se="o-form-error-container"] [data-se="callout"] > h3';
 const FORM_INFOBOX_ERROR_TITLE_V3 = '[data-se="callout"] h2';
 const CANT_SCAN_BUTTON_TEXT = 'Setup without scanning a QR code.';
+const SETUP_OV_BUTTON_TEXT = 'Set up Okta Verify';
+const SETUP_ON_ANOTHER_MOBILE_DEVICE_TEXT = 'Or set up Okta Verify on another mobile device';
```

Comment:
Since we updated the link to make the proper text a link value, just need to target the link text now.
```suggestion
const SETUP_ON_ANOTHER_MOBILE_DEVICE_TEXT = 'set up Okta Verify on another mobile device';
```